src/Extension.php

```diff
@@ -50,9 +50,12 @@ private function parameterAsArray(ParameterCollection $parameters, string $name)
             return null;
         }
 
-        return array_map(
-            fn(string $value): string => trim($value),
-            explode($value, ",")
-        );
+        $data = [];
+        foreach(explode(',', $value) as $item)
+        {
+            $data[] = trim($item);
+        }
+
+        return $data;
```

Comment:
Thank you for the contribution @daycry. Can you please provide some examples where the changed code will produce different output compared to the existing one?

Reply:
Ah, i see the issue now. The order of the `explode` arguments is wrong in the existing code. Can you please update the PR to only fix the order please?

Reply:
Done